stdlib/universe/fill.go

```diff
@@ -251,11 +260,16 @@ func (t *fillTransformation) Finish(id execute.DatasetID, err error) {
 }
 
 func (t *fillTransformation) fillTable(w *table.StreamWriter, cr flux.ColReader, colIdx int, fillValue *interface{}) error {
-	if cr.Len() == 0 {
+	crColsLen := len(cr.Cols())
+	wColsLen := len(w.Cols())
+	crLen := cr.Len()
+	if crLen == 0 || crColsLen == 0 || wColsLen == 0 {
```

Comment:
I'm not sure why this condition changed. Is there a test case that exercised this?

Reply:
This is just for an extra safety check, if for any reason read or write cols length is 0, code should return immediately.

Reply:
I think this will end up being confusing. It's not possible for either of these to be zero so there's no way to test it. Can you revert this condition back? If they were zero, I believe the below code would still work anyway.

Reply:
one of the checks was already there. Should I remove that as well ?
https://github.com/influxdata/flux/blob/1a84fac74554ebb7bd1674c8671ec96049de0055/stdlib/universe/fill.go#L254

Reply:
No that check is possible and important. It's possible for a table to be empty. It's just not possible for it to have no columns.